user_groups/user_groups.md

```diff
@@ -298,9 +299,62 @@ Response
 ```
 ---
 
-## Add or update user group permission for a given resource
+## Add user group permission for a given resource
 
-Add or update resource permission of a user group.
+Add resource permission of a user group.
+
+```apacheconfig
+POST /v1/usergroups/{userGroupId}/permissions
+```
+
+#### Path parameters
+
+| Name | Type | Description |
+| ------------- |------------- | -------------|
+| userGroupId | `string` | The user group id to request. |
+| resourceId | `string` | The resource id to add. |
+
+#### Query parameters
+The query parameters are not expected.
+
+#### Request body
+```json
+{
+  "resourceId": "p-2343",
+  "role": "reader"
+}
+```
+
+#### Response body
+The successful response contains a [Permission Role Type](../permissions/permissions_schemas.md/#permission-role-type).
```

Comment:
its permission role, not permission role type. Make sure you link the right one

Reply:
updated